src/Monitors/IMUDownlinkReportMonitor.cpp

```diff
@@ -39,11 +39,9 @@ void IMUDownlinkReportMonitor::create_imu_downlink_report(int fragment_number)
     }
 
     // add actual gyro content to imu report
-    int z = 0;
```

Comment:
66 should be a constant- I see this was also a problem in the original.

Reply:
Everything else looks good- once there is a constant for the content length of gyro data this should be good.